Assets/Scripts/Player/Movement/MouseLook.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MouseLook : MonoBehaviour
+{
+    [SerializeField] private float mouseSensitivity;
+    [SerializeField] private Transform playerBody;
+    [SerializeField] private float xRotation;
+    
+    private float _mouseX;
+    private float _mouseY;
+
+    private void Start()
+    {
+        Cursor.lockState = CursorLockMode.Locked;
+    }
+
+    private void Update()
+    {
+        _mouseX = Input.GetAxis("Mouse X") * mouseSensitivity * Time.deltaTime;
+        _mouseY = Input.GetAxis("Mouse Y") * mouseSensitivity * Time.deltaTime;
+
+        xRotation -= _mouseY;
+        xRotation = Mathf.Clamp(xRotation, -90f, 90f);
```

Comment:
Kan je de 90 veranderen naar een private const, want dan hebben we minder random nummers.